src/trace-server.ts

```diff
@@ -2,6 +2,7 @@ import { ChildProcess, spawn } from 'child_process';
 import { TspClient } from 'tsp-typescript-client/lib/protocol/tsp-client';
 import treeKill from 'tree-kill';
 import * as vscode from 'vscode';
+import { parseArgs } from './argparse';
```

Comment:
It might be prudent to add "argparse" as a  production dependency - I see it's indirectly pulled in the development environment, but I am not sure it will be present in the packaged extension?

Reply:
Never mind - `argparse` used here is local, not related to the devDependency of that name, pulled in the dev environment :)